File: LeetCode/RemoveDublicate.js
let removeDublicate = function(nums){
	let i = 0;
	for(let j = 0; j<nums.length; j++){
		if(nums[i] != nums[j]){
			i++;
			nums[i] = nums[j];
		}
	}
	return i + 1;
};

const readline = require('readline');
const read =  readline.createInterface({
	input: process.stdin,
	output: process.stdout
});

read.question('', function(array){
	let temp = array.split(" ");
	let arr = temp.map(function(value){
		return Number(value);
	});
	let result = removeDublicate(arr);
	console.log(result);
});

read.on("close", function(){
	process.exit(0);
});

File: LeetCode/RemoveDublicates/RemoveDublicate.py
def removeDublicate(arr):
	i = 0
	for j in range(1, len(arr)):
		if arr[i] != arr[j]:
			i += 1
			arr[i] = arr[j]

	return i + 1


arr = list(map(int, input().split()))
res = removeDublicate(arr)
print(res)